src/Components/Trainers/index.js

```diff
@@ -1,11 +1,57 @@
 import styles from './trainers.module.css';
+import Table from './Table';
+/* import Modal from './Modal'; */
+import { useEffect, useState } from 'react';
+
+const Trainers = () => {
+  const [trainers, setTrainers] = useState([]);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [deleteModalMessage, setDeleteModalMessage] = useState('');
+
+  const getTrainers = async () => {
+    const response = await fetch('https://nai-megarocket-server.vercel.app/api/trainers');
+    const data = await response.json();
+    setTrainers(data.data);
+  };
+  useEffect(() => {
+    getTrainers();
+  }, []);
+  const DeleteModal = () => {
+    const handleCloseModal = () => {
+      setShowDeleteModal(false);
+    };
+
+    return (
+      <div className={styles.deleteModal}>
+        <p>{deleteModalMessage}</p>
+        <button onClick={handleCloseModal}>Close</button>
+      </div>
+    );
+  };
+  const deleteItem = async (id) => {
+    try {
+      await fetch(`https://nai-megarocket-server.vercel.app/api/trainers/${id}`, {
+        method: 'DELETE'
+      });
+
```

Comment:
We should not leave empty lines

Reply:
Thank you for your review. The changes have been completed and it's ready!